urllib3/response.py

```diff
@@ -515,6 +515,13 @@ def readinto(self, b):
             b[:len(temp)] = temp
             return len(temp)
 
+    def supports_chunked_reads(self):
+        """
+        Checks if the underlying file-like object looks like a
+        httplib.HTTPResponse object.
```

Comment:
Can we elaborate on this docstring a little bit? The docstring and the name seem to be unrelated. Basically we can just use some of the terminology from `BodyNotHttplibCompatible`.